packages/ra-data-simple-rest/src/index.spec.ts

```diff
@@ -120,5 +193,20 @@ describe('Data Simple REST Client', () => {
                 }
             );
         });
+        it('should escape id in path', async () => {
+            const httpClient = jest
+                .fn()
+                .mockResolvedValue({ json: { id: 'Post#123' } });
```

Comment:
This isn't a valid `updateMany` response

Reply:
Fixing.